mcenter_cli/parallelm/mcenter_cli/mcli.py

```diff
@@ -0,0 +1,100 @@
+import sys
+import logging
+
+from cliff.app import App
+from cliff.commandmanager import CommandManager
+
+from parallelm.mcenter_cli import version
+from parallelm.mcenter_client.mcenter_client import MCenterClient
+
+
+class MCli(App):
+
+    DEFAULT_LOG_LEVEL = "error"
+
+    def __init__(self, prog_name):
+
+        super(MCli, self).__init__(
+            description='mcli - MCenter CLI',
+            version=version,
+            command_manager=CommandManager('mcenter.cli'),
+            deferred_help=True,
+        )
+
+        self._prog_name = prog_name
+        self._options = None
+        self.mclient = None
+        self._logging_level = logging.ERROR
+        self._commands = {}
+        self._example_text = '''Examples:
+            {prog_name} --server localhost mlapp-load /opt/mlapps/regression-prod-v32 
+            {prog_name} mlapp-download my-ml-app  /opt/mlapps/my-ml-app-new
+            {prog_name} mlapp-delete my-ml-app
+            {prog_name} get-action-log --object=MLApp,Model --action=Stop,Upload --username=admin,MLOPSUser --status=success
+            '''.format(prog_name=prog_name)
+        self._argv_0 = sys.argv[0]
+
+        # self._init_parser(logo)
```

Comment:
This need this commented out code -- lines 37 to 46?

Reply:
done